File: source/java/com/glympse/java/hal/GThread.java
//------------------------------------------------------------------------------
//
// Copyright (c) 2014 Glympse Inc.  All rights reserved.
//
//------------------------------------------------------------------------------

package com.glympse.java.hal;

import java.util.*;
import com.glympse.java.core.*;
import com.glympse.java.hal.*;
    
interface Thread extends GCommon
{
    public void start();
    
    public void join();    
};
    
typedef Thread  Thread;
    

File: source/csharp/glympse/hal/GThread.cs
//------------------------------------------------------------------------------
//
// Copyright (c) 2014 Glympse Inc.  All rights reserved.
//
//------------------------------------------------------------------------------

using System;
using System.Collections;
using System.Collections.Generic;
using System.Text;
using System.Threading;

namespace Glympse 
{
    
interface Thread : GCommon
{
     void start();
    
     void join();    
};
    
typedef Thread  Thread;
    
}

